gen/io/github/intellij/dlanguage/psi/DLanguageAsmInstruction.java

```diff
@@ -26,6 +26,6 @@ public interface DLanguageAsmInstruction extends PsiElement {
     public PsiElement getOP_COLON();
 
     @Nullable
-    public PsiElement getSEMICOLON();
+    public PsiElement getOP_SCOLON();
```

Comment:
re the naming convention; why move away from using the term `SEMICOLON`? That seems to be the convention Jetbrains use. If it makes sense to prefix it then how about using `OP_SEMICOLON`.  I won't block merging on this of course

Reply:
I changed the name to call it as the underlying psi element is named. This method is never called, so I can named it `getSEMICOLON()`, but the element it return is `OP_SCOLON` because it’s how it’s named in the DLangTokens.

I agree to rename the token `OP_SEMICOLON` or directly `SEMICOLON`